test/eoftests/eoftests_runner.cpp

```diff
@@ -0,0 +1,61 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2022 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "../utils/utils.hpp"
+#include "eoftests.hpp"
+#include <evmc/evmc.hpp>
+#include <evmone/eof.hpp>
+#include <gtest/gtest.h>
+
+
+namespace evmone::test
+{
+namespace json = nlohmann;
+void run_eof_test(std::istream& input)
+{
+    const EOFTests test = json::json::parse(input).get<EOFTests>();
+    for (const auto& [name, cases] : test.cases)
+    {
+        for (const auto& expectation : cases.expectations)
+        {
+            const EOFValidationError result = evmone::validate_eof(expectation.rev, cases.code);
+            const bool b_result = (result == EOFValidationError::success);
+            EXPECT_EQ(b_result, expectation.result) << expectation.rev << " " << hex(cases.code);
```

Comment:
Also useful to output case name, since you happen to have it
```suggestion
            EXPECT_EQ(b_result, expectation.result) << name << " " << expectation.rev << " " << hex(cases.code);
```